.changes/2.15.25.json

```diff
@@ -35,7 +35,7 @@
         {
             "type": "bugfix",
             "category": "AWS SDK for Java v2",
-            "contributor": "@frosforever",
+            "contributor": "[@frosforever](https://github.com/frosforever)",
```

Comment:
Doing the formatting in the property value itself makes changelog generation less flexible; e.g. what if we want to just collect a list of contributors without the link? We should do the formatting of the link at the generation step.

Reply:
Fair enough. Will update